pkg/logger/logger.go

```diff
@@ -76,7 +76,12 @@ func (l *SugarLogger) With(args ...interface{}) *SugarLogger {
 	}
 }
 
-func (l *SugarLogger) SetLogLevel(level string) error {
+func (l *SugarLogger) IsDebug() bool {
+	return l.conf.Level.Level() == zap.DebugLevel
+}
+
+// setLogLevel is used only for tests
+func (l *SugarLogger) setLogLevel(level string) error {
```

Comment:
For now, it is used only for tests, but we might add operation endpoints for changing it dynamically at runtime, so lets keep it public.